hosts/ficolobuild/configuration.nix

```diff
@@ -36,4 +36,10 @@
   # Use the systemd-boot EFI boot loader.
   boot.loader.systemd-boot.enable = true;
   boot.loader.efi.canTouchEfiVariables = true;
+
+  # Connect hosts in the same network.
+  networking.extraHosts = "
+    172.18.20.102 vedenemo.dev # for fetching Gala app sources
+    172.18.20.109 cache.vedenemo.dev # Binary cache
```

Comment:
Why you still need cache.vedenemo.dev?

Reply:
cache.vedenemo.dev is still in the substituters list included with hosts/common.nix to x86 builder setup. I did not start to split common.nix yet as I found out that when these builders are used as remote builders, those pre-defined substituters are not used by default. You still need to explicitly instruct remote build machines to use their own binary substitutes:
https://nixos.org/manual/nix/stable/command-ref/conf-file.html#conf-builders-use-substitutes
So, this is enabling usage of that binary cache but not forcing of its usage.